tpgtools/api/cloudbuildv2/samples/ghe_complete.connection.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "{{connection}}",
+  "githubEnterpriseConfig": {
+    "hostUri": "https://ghe.proctor-staging-test.com",
+    "privateKeySecretVersion":"projects/1033762898806/secrets/gcbrepos-dcltest-private-key/versions/1",
```

Comment:
What project do the referenced secrets live in? Are they publicly accessible?

We're likely to need to do a permissions grant within the test, create the secrets we reference, or similar.

Reply:
Moved to a separate PR: https://github.com/GoogleCloudPlatform/magic-modules/pull/7157

Adjusted to use a separate project as agreed offline.